include/intx/intx.hpp

```diff
@@ -1608,16 +1608,19 @@ constexpr uint256 mulmod(const uint256& x, const uint256& y, const uint256& mod)
     return udivrem(umul(x, y), mod).rem;
 }
 
+#define INTX_JOIN2(X, Y) X##Y
+#define INTX_JOIN1(X, Y) INTX_JOIN2(X, Y)
```

Comment:
Do we need two levels of these `JOIN` macros?

Reply:
> Do we need two levels of these `JOIN` macros?

I didn't make it that way because I was bored :) At first I took whatever was from boost, and their join is 3-step nowadays:
```c++
#define BOOST_JOIN(X, Y) BOOST_DO_JOIN(X, Y)
#define BOOST_DO_JOIN(X, Y) BOOST_DO_JOIN2(X,Y)
#define BOOST_DO_JOIN2(X, Y) X##Y
```
whatever I did, I couldn't make it with 1 or 2 defines. I was already convinced that there was no way to avoid the warning in the first place, but then figured out what makes msvc happy.

Reply:
did a bit more tinkering, and just got rid of one join